.gitbook/contents/notices/fraud-proofs.md

```diff
@@ -0,0 +1,27 @@
+# Prepare for Fraud Proofs on Boba Sepolia and Boba BNB Testnet 
+
+Fraud proofs are important for making L2 more decentrailized, allowing more community involvement in validating the L2 state. These changes allow anyone to withdraw funds from Boba Layer 2 to L1without relying on any centralized actors.
```

Comment:
`L1without` is run together.

Perhaps though something like

These changes allow anyone to submit or challenge a state root, an essential part of the process used for withdrawing funds from L2 to L1.

Reply:
Fixed